generator/templates/Authorisation/src/locales/nl/authorisation.json

```diff
@@ -1,7 +1,12 @@
 {
     "login": {
         "title": "Inloggen",
-        "successMessage": "Je bent ingelogd, we sturen je nu door."
+        "successMessage": "Je bent ingelogd, we sturen je nu door.",
+        "messages": {
+            "invitationAcceptSuccess": "Je account is geactiveerd, je kunt nu inloggen.",
+            "passwordResetSuccess": "Je wachtwoord is veranderd, je kunt nu inloggen.",
+            "registrationVerify": "Je registratie is voltooid, je kunt nu inloggen."
+        }
```

Comment:
You forgot to remove these messages below (i quess where you copied them from?)
for example line 23

Reply:
Really weird, when opening this branch local they were removed..